### PR TITLE
Greg.bowman/api 5948 add monthly upload size

### PR DESCRIPTION
**Pull request by gpbowman-git**
## Description of change

Added in upload size metrics and refactored code

## Original issue(s)
https://vajira.max.gov/browse/API-5948

### CODE REVIEW

**Review by cshupp1 — APPROVED** [review]

File: config/sidekiq_scheduler.yml [hunk 1]
```diff
@@ -120,7 +120,7 @@ VBADocuments::ReportUnsuccessfulSubmissions:
   description: "Weekly report of unsuccessful benefits intake submissions"
 
 VBADocuments::ReportMonthlySubmissions:
-  cron: "4 3 2 7 * * America/New_York"
+  cron: "4 3 2 15 * * America/New_York"
```
Comment (ericboehs) [inline config/sidekiq_scheduler.yml:123]: I'm not familiar with this form of cron. Could you explain when you want this to run in plain English?

**Comment by LindseySaari** [comment]
Would you be able to pull the fixtures into a separate PR that gets merged before the changes referencing the fixtures?

File: config/sidekiq_scheduler.yml (hunk 1 above)
Comment (gpbowman-git) [reply config/sidekiq_scheduler.yml:123]: this will run at 02:03:04 on the 15th day of each month.

Comment (ericboehs) [reply config/sidekiq_scheduler.yml:123]: So that explains the first 5 (4 3 2 15 *). What's the 6th place (*) for?

Comment (gpbowman-git) [reply config/sidekiq_scheduler.yml:123]: I believe the 5th position is for the month and the sixth is for the day of the week.

File: modules/vba_documents/app/mailers/vba_documents/monthly_report_mailer.rb [hunk 2]
```diff
@@ -18,10 +18,12 @@ def self.fetch_recipients
 
     # rubocop:disable Metrics/MethodLength
     # rubocop:disable Metrics/ParameterLists
-    def build(monthly_totals, summary, still_processing, avg_processing_time, last_month_start, last_month_end)
+    def build(monthly_totals, summary, still_processing, still_success,
```
Comment (LindseySaari) [inline modules/vba_documents/app/mailers/vba_documents/monthly_report_mailer.rb:21]: Looks like there's an extra newline here

File: modules/vba_documents/app/workers/vba_documents/report_monthly_submissions.rb [hunk 3]
```diff
@@ -2,94 +2,12 @@
...
-    	avg(date_part('epoch', a.updated_at)::bigint -
-        date_part('epoch', a.created_at)::bigint)::integer as avg_time_secs
-      from vba_documents_upload_submissions a
-      where a.status = 'vbms'
-      and   a.created_at < $1
-      group by 1,2
-      order by 1 desc, 2 desc
-      limit 12
-    "
+    extend VBADocuments::SQLSupport
```
Comment (LindseySaari) [inline modules/vba_documents/app/workers/vba_documents/report_monthly_submissions.rb:10]: Nice cleanup 👍

**Review by LindseySaari — APPROVED** [review]

File: modules/vba_documents/app/mailers/vba_documents/monthly_report_mailer.rb (hunk 2 above)
Comment (gpbowman-git) [reply modules/vba_documents/app/mailers/vba_documents/monthly_report_mailer.rb:21]: that line break is there to fix the linter line length error

**Review by LindseySaari — APPROVED** [review]
